contrib/babelfishpg_common/src/collation.h

```diff
@@ -83,7 +83,7 @@ typedef struct collation_callbacks
 	/* Function pointers set up by the plugin */
 	char	   *(*EncodingConversion) (const char *s, int len, int src_encoding, int dest_encoding, int *encodedByteLen);
 
-	Oid			(*get_server_collation_oid_internal) (bool missingOk);
```

Comment:
So when database and server collation are different, which one will be the answer ?